resources/public/api/conf/1.0/application.raml

```diff
@@ -118,92 +119,92 @@ uses:
                 type: types.errorResponse
                 examples:
                   Forbidden:
-                    description: The organisation's details provided did not match with HMRC's records
+                    description: "The organisation's details provided did not match with HMRC's records"
                     value:
                       code: MATCHING_FAILED
-  /details/crn:
-    /{matchId}:
+  /matching/paye/{matchId}:
       uriParameters:
         matchId:
-          description: A short-lived UUID, typically valid for 5 hours
+          description: "A short-lived UUID, typically valid for 5 hours"
           type: string
           example: 57072660-1df9-4aeb-b4ea-cd2d7f96e430
       get:
-        displayName: CRN Details
-        description: This endpoint takes a matchId and returns corporation tax information for the company
+        displayName: "Get matched organisation's PAYE details"
+        description: "This endpoint takes a matchId and returns corporation tax information for the company"
         is: [headers.acceptHeader, headers.contentHeader]
         (annotations.scope): "assigned"
         securedBy: [ sec.oauth_2_0: { scopes: [ "assigned" ] } ]
+        (annotations.sandboxData): !include sandbox/paye-sandbox-data.md
         headers:
           Authorization:
-            description: OAuth 2.0 Bearer Token with the read:organisations-matching scope.
+            description: "OAuth 2.0 Bearer Token with the read:organisations-matching scope."
             type: string
             required: true
             example: Bearer <BEARER_TOKEN>
         queryParameters:
           fromYear:
             displayName: From Year
             type: number
-            description: The earliest Tax Year from which you would like details
+            description: "The earliest Tax Year from which you would like details"
             example: 2017
             required: true
           toYear:
             displayName: To Year
             type: number
-            description: The latest Tax Year from which you would like details, if not provided then current year will be used
+            description: "The latest Tax Year from which you would like details, if not provided then current year will be used"
             example: 2019
             required: false
         responses:
           200:
-            description: The company's Corporation Tax details
+            description: "The company's Corporation Tax details"
             body:
               application/json:
-                type: !include schemas/crn-details-response.json
-                example: !include examples/crn-details-response-example.json
+                type: !include schemas/paye-details-response.json
```

Comment:
thi payload seems wrong, running locally shows "taxpayer type and taxpayer name" whereas matrix shows employerName.